src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt

```diff
@@ -214,6 +227,25 @@ class CaptureFarmingGear {
                 storage.anitaUpgrade = level
             }
         }
+        if (event.inventoryName.contains("Visitor Milestones")) {
```

Comment:
Please use early reuturn over if

Reply:
Why? There are a bunch of other similar `if`s in this function, it isn't the only one. It would look weird to only early return for the last one.

Reply:
Same goes for all other if checks that have a longer block inside. Multiple additions control flows just increase the intendation size unnecessary, in my mind

Reply:
You can't exactly do that when there are multiple inventory names being tested. If you early return from one, it won't check the others.

Reply:
I looked over the whole function and you are right, one single early return feels wrong here, especially since it's not "early". Another idean I just had was to replace the chained ifs with one select block and extract each case into a function. Would increase the class size but split up a bigger function into smaller logical steps.